### PR TITLE
feat(settings): export breakpoints from settings

### PR DESCRIPTION
**Pull request by alanfriedman**
just blindly merge this don't ask any questions please, thank you

### CODE REVIEW

File: packages/core/src/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import './global/js/polyfills';
 import boot from './global/js/boot';
+import settingsObj from './global/js/settings';
```
Comment (alanfriedman) [inline packages/core/src/index.js]: @adamraider Not sure a better way to do this. Jest was complaining about default exports

Comment (adamraider) [reply packages/core/src/index.js]: `export {default as name1} from '...';`
try that

File: packages/core/src/index.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 
 boot();
 
 export * from './components';
+export const settings = settingsObj;
```
Comment (adamraider) [inline packages/core/src/index.js]:
```suggestion
export { default as settings } from './global/js/settings';
```

File: packages/core/src/index.js [hunk 3]
```diff
@@ -4,3 +4,4 @@ import boot from './global/js/boot';
 boot();
 
 export * from './components';
+export { default as settings } from './global/js/settings';
```
Comment (alanfriedman) [inline packages/core/src/index.js]: omg it works

File: packages/core/src/index.js (hunk 2 above)
Comment (alanfriedman) [reply packages/core/src/index.js]: I reject your suggestion and commit it myself

Comment (adamraider) [reply packages/core/src/index.js]: :(

**Comment by adamraider** [comment]
@allcontributors[bot] please add @alanfriedman as a developer

**Comment by adamraider** [comment]
@allcontributors[bot] please add @alanfriedman for code

**Comment by alanfriedman** [comment]
This bot is insane

**Comment by alanfriedman** [comment]
you ok bot?

**Comment by alanfriedman** [comment]
@allcontributors[bot] hey

**Comment by alanfriedman** [comment]
@allcontributors[bot] just saying hey

**Comment by alanfriedman** [comment]
@allcontributors[bot] please just drop it

**Comment by alanfriedman** [comment]
@adamraider tell bot to leave me alone

**Pull request merged**